guides/common/modules/proc_creating-an-organization.adoc

```diff
@@ -11,6 +11,15 @@ To use the CLI instead of the {ProjectWebUI}, see xref:cli-creating-an-organizat
 . In the *Label* field, enter a unique identifier for the organization.
 This is used for creating and mapping certain assets, such as directories for content storage.
 Use letters, numbers, underscores, and dashes, but no spaces.
+ifdef::satellite[]
```

Comment:
I am following internal discussions about SCA and public discussions within the community ([[RFC] Making things easier when working with custom products & Simple Content Access (SCA)](https://community.theforeman.org/t/rfc-making-things-easier-when-working-with-custom-products-simple-content-access-sca/31334) in Foreman Community Forum). Are we sure to recommend upstream users to always use SCA? Is there a usecase where it might make sense for them to also disable SCA?

I think the "Satellite only" makes sense if Foreman 3.5+ users *cannot* disable SCA (anymore).

cc @jeremylenz @ianballou @m-bucher

Reply:
I agree. For the time being, entitlement mode remains supported both upstream and downstream. So this section should be included in both doc sets.

Reply:
If I understand this correctly, I should leave the contents  of the note but make it visible not only for satellite?

Reply:
Right, I'd just remove the `ifdef::satellite[]`.

Reply:
Got it, thanks!

Reply:
> Are we sure to recommend upstream users to always use SCA? Is there a usecase where it might make sense for them to also disable SCA?

I am not aware of this usecase; however; we provide the user with the option to disable SCA if they know what they're doing. I think it is okay to generally recommend the new method over the old one, but then again I am not a dev.